ambari-server/src/main/resources/stacks/BIGTOP/3.2.0/services/SPARK/service_advisor.py

```diff
@@ -189,18 +189,11 @@ def recommendSpark2ConfigurationsFromHDP25(self, configurations, clusterData, se
     :type services dict
     :type hosts dict
     """
-    putSparkProperty = self.putProperty(configurations, "spark2-defaults", services)
-    putSparkThriftSparkConf = self.putProperty(configurations, "spark2-thrift-sparkconf", services)
-
-    spark_queue = self.recommendYarnQueue(services, "spark2-defaults", "spark.yarn.queue")
+    putSparkProperty = self.putProperty(configurations, "spark-defaults", services)
+    spark_queue = self.recommendYarnQueue(services, "spark-defaults", "spark.yarn.queue")
     if spark_queue is not None:
       putSparkProperty("spark.yarn.queue", spark_queue)
 
-    spark_thrift_queue = self.recommendYarnQueue(services, "spark2-thrift-sparkconf", "spark.yarn.queue")
-    if spark_thrift_queue is not None:
-      putSparkThriftSparkConf("spark.yarn.queue", spark_thrift_queue)
-
-
   def recommendSPARK2ConfigurationsFromHDP26(self, configurations, clusterData, services, hosts):
```

Comment:
The spark version in bigtop 3.2.0 is 3.2.3,  would it be better if we change  "spark2"  to "spark" in service advsior.py?